bindings/python/pymongoarrow/schema.py

```diff
@@ -68,6 +68,12 @@ def _normalize_sequence(sequence):
                 normed[fname] = _normalize_typeid(ftype, fname)
         return normed
 
+    def _get_projection(self):
+        projection = {'_id': False}
```

Comment:
We never seem to test that `_id` is omitted by default. Should we add a test for _get_projection in test_schema.py?

Reply:
Instead of unit testing that private method, I have added integration tests that check the `_id` field is omitted from server responses.